style.css

```diff
@@ -193,7 +193,6 @@ h1 {
 }
 
 /* Works Section */
```

Comment:
Good job with the design. 👌 But let's add some padding like on the Figma template. 
<img width="354" alt="Screenshot 2023-02-01 at 11 06 10 PM" src="https://user-images.githubusercontent.com/25887253/216153111-38f6b3dc-008a-444f-b658-184f735c3e82.png">
<img width="354" alt="Screenshot 2023-02-01 at 11 06 18 PM" src="https://user-images.githubusercontent.com/25887253/216153118-f5bca3cf-3c19-4401-bfc9-0568585f40b3.png">